crates/core/Cargo.toml

```diff
@@ -23,4 +23,4 @@ tree-sitter.workspace = true
 default = ["regex"]
 
 [dev-dependencies]
-tree-sitter-typescript = "0.21.1"
\ No newline at end of file
+tree-sitter-typescript = { package = "tree-sitter-typescript-codemod", version = "0.25.0-codemod.1"}
```

Comment:
I just manually rebuilt `tree-sitter-typescript` with the latest version of the ts cli.